### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by chloewlin**
Setting up a PR for my upcoming translation in Accessibility.

### CODE REVIEW

**Comment by neighborhood999** [comment]
Hi @chloewlin, is it still in progress?

**Comment by chloewlin** [comment]
@neighborhood999 Yes! This is a long article. I will change the label and ping you when it's ready.

File: content/docs/accessibility.md [hunk 1]
```diff
@@ -40,15 +40,14 @@ Note that all `aria-*` HTML attributes are fully supported in JSX. Whereas most
...
-Semantic HTML is the foundation of accessibility in a web application. Using the various HTML elements to reinforce the meaning of information
-in our websites will often give us accessibility for free.
+Semantic HTML 是無障礙網頁應用程式的基礎。使用不同的 HTML element 來加強網站中資訊的意義可以在不用花費的情況下讓所有人造訪你的網站。
 
-- [MDN HTML elements reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
+- [MDN HTML element 參考](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)
 
-Sometimes we break HTML semantics when we add `<div>` elements to our JSX to make our React code work, especially when working with lists (`<ol>`, `<ul>` and `<dl>`) and the HTML `<table>`.
-In these cases we should rather use [React Fragments](/docs/fragments.html) to group together multiple elements.
+有時當我們增加 `<div>` element 到 JSX 以使我們的 React 程式運作時，我們會違反 HTML 的語法規定，尤其是當我們在處理列表（`<ol>`，`<ul>` 和 `<dl>`）以及 HTML 表格 `<table>` 的時候。
```
Comment (neighborhood999) [inline content/docs/accessibility.md]:
```suggestion
有時候當我們新增 `<div>` element 到 JSX 讓 React 程式可以運作時，我們會違反 HTML 的語義，尤特別是在當我們處理列表（`<ol>`，`<ul>` 和 `<dl>`）以及 HTML 表格 `<table>` 的時候。
```

File: content/docs/accessibility.md [hunk 2]
```diff
@@ -104,60 +103,60 @@ function ListItem({ item }) {
...
-Keyboard focus refers to the current element in the DOM that is selected to accept input from the keyboard. We see it everywhere as a focus outline similar to that shown in the following image:
+鍵盤焦點指的是目前在 DOM 中的 element 被選取以接受來自鍵盤的輸入。我們到處可以見到類似下面這張圖示內的焦點輪廓：
 
 <img src="../images/docs/keyboard-focus.png" alt="Blue keyboard focus outline around a selected link." />
 
-Only ever use CSS that removes this outline, for example by setting `outline: 0`, if you are replacing it with another focus outline implementation.
+如果你打算用另一種方式做焦點輪廓，請使用 CSS 來移除這個輪廓，例如設定 `outline: 0`。
 
-### Mechanisms to skip to desired content {#mechanisms-to-skip-to-desired-content}
+### 跳到指定內容的方式 {#mechanisms-to-skip-to-desired-content}
```
Comment (neighborhood999) [inline content/docs/accessibility.md]:
```suggestion
### 跳到指定內容的機制 {#mechanisms-to-skip-to-desired-content}
```

**Pull request merged**